bisection.py

```diff
@@ -220,26 +239,31 @@ class TorchBench:
     timelimit: int # timeout limit in minutes
     workdir: str
     devbig: str
+    models: List[str]
     torch_src: TorchSource
 
     def __init__(self, srcpath: str,
                  torch_src: TorchSource,
                  timelimit: int,
                  workdir: str,
                  devbig: str,
-                 branch: str = "0.1"):
+                 branch: str = "master"):
         self.srcpath = srcpath
         self.torch_src = torch_src
         self.timelimit = timelimit
         self.workdir = workdir
         self.devbig = devbig
         self.branch = branch
+        self.models = list()
 
     def prep(self) -> bool:
         # Verify the code in srcpath is pytorch/benchmark
         repo_origin_url = gitutils.get_git_origin(self.srcpath)
         if not repo_origin_url == TORCHBENCH_GITREPO:
             print(f"WARNING: Unmatched repo origin url: {repo_origin_url} with standard {TORCHBENCH_GITREPO}")
+        # get list of models
+        self.models = [ model for model in os.listdir(os.path.join(self.srcpath, "torchbenchmark", "models"))
```

Comment:
any reason to do this instead of importing 'list_models' function from torchbenchmark?

Reply:
When this function is called, the pytorch/torchvision/torchtext packages are not built and installed yet (they are built later during bisection/abtesting). Therefore, the list_models function doesn't work at this moment.